src/components/SnapshotDetails/__tests__/EnvironmentProvisionErrorAlert.spec.tsx

```diff
@@ -37,7 +37,8 @@ describe('EnvironmentProvisionErrorAlert', () => {
     expect(screen.getByTestId(/env-provision-err-alert/)).toBeInTheDocument();
     expect(screen.getByText('app-sample-go-basic-enterprise-contract')).toBeInTheDocument();
     expect(screen.getByText(/Snapshot failed to deploy/)).toBeInTheDocument();
-    expect(screen.getByText(/Sep 19, 2023/)).toBeInTheDocument();
+    // date in Sep 19, 2023 format or in 19. 9. 2023 format
```

Comment:
Fixed test to work on both time formats.

Reply:
We use Sep 19, 2023 format everywhere. Why do you need 19.9.2023 format?

Reply:
When I run it locally it expects 19.9.2023 format 😄. So I rewrote it to be both.

Reply:
It seems like this change cannot possibly be logically related to the change in the PR description.

My two cents: you should separate it into a separate commit and separate PR to be considered independently, otherwise this change will be hung up on wonder and debate over this unexpected extra thing.

(You probably need adjust some of your locale environment variables when running the test to get the original expected date format.)

Reply:
I will do it that way. Reverted to original form.

Reply:
Thanks! I'll resolve this comment, but I'll let others provide final review on the overall change (those who know what they're doing more with this codebase than I do!)